kubernetes/README.md

```diff
@@ -4,7 +4,7 @@ No description provided (generated by Swagger Codegen https://github.com/swagger
 This Python package is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
 
 - API version: v1.12.2
-- Package version: 8.0.0-snapshot
```

Comment:
`README.md` and `client/configuration.py` had merge conflicts between master and release-8.0 branch. I kept changes from master, therefore here it shows `8.0.0-snapshot`.